<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompactWindow</class>
 <widget class="QWidget" name="CompactWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>140</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>140</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mouse Control</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="recordButton">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string></string>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="playButton">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string></string>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="statusLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ready</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
